spec/requests/invites_controller_spec.rb

```diff
@@ -203,6 +203,52 @@
     end
   end
 
+  context '#create_multiple_use_invite_link' do
+    it 'requires you to be logged in' do
+      post "/invites/invite-link.json", params: {
+        max_redemption_allowed: 5
+      }
+      expect(response.status).to eq(500)
```

Comment:
500 ... we don't want 500 errors

https://github.com/discourse/discourse/blob/a2d939608d5b3adc7f037dc0ca38e7d1f6b895f6/spec/requests/topics_controller_spec.rb#L481-L481

Reply:
Oops, fixed!